src/einsteinpy/coordinates/utils.py

```diff
@@ -0,0 +1,178 @@
+import numpy as np
+
+from ..ijit import jit
+
+
+def cartesian_to_spherical_fast(
+    x, y, z, v_x=None, v_y=None, v_z=None, velocities_provided=False
```

Comment:
if this function is calling `cartesian_to_spherical()` & `cartesian_to_spherical_novel()`, shouldn't this function be defined below the other two. I know it doesn't matter in python but anyways, it's a commonly accepted practice in programming.

Reply:
> it's a commonly accepted practice in programming

It's good to follow certain conventions, although!

Reply:
We will fix it. Don't worry